public/wanna-isotope-shortcode.php

```diff
@@ -167,20 +210,26 @@ public function shortcode_isotope( $atts ) {
                 }
 
             endwhile; ?>
-            </ul> 
+            </ul>
+
+            <?php
+            if('yes'==$this->pagination){
```

Comment:
Can you fix brace style
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style